addons/website_blog/data/website_blog_data.xml

```diff
@@ -45,7 +31,7 @@
         <record id="action_open_website" model="ir.actions.act_url">
             <field name="name">Website Blogs</field>
             <field name="target">self</field>
-            <field name="url" eval="'/blog/'+str(ref('website_blog.blog_blog_1'))"/>
+            <field name="url" eval="'/blog/'"/>
```

Comment:
in this case, be sure to handle the case, where if only oneblog 
/blogs redirect to this blog (like for forum e.g.)

Reply:
Is this enough ?

https://github.com/odoo/odoo/blob/12194d4aec2dc0fc878e42308db7a4792d6a95a6/addons/website_blog/controllers/main.py#L56

Reply:
yes :+1: 

but /blog instead of /blog/ is better ;)